.github/workflows/catalogs.yml

```diff
@@ -168,6 +168,27 @@ jobs:
           path: nebius.csv
           retention-days: 1
 
+  catalog-cudo:
+    name: Collect Cudo catalog
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+      - name: Install dependencies
+        run: |
+          pip install pip -U
+          pip install -e '.[cudo]'
+      - name: Collect catalog
+        working-directory: src
+        run: python -m gpuhunt cudo --output ../cudo.csv
+      - uses: actions/upload-artifact@v3
```

Comment:
Please, update this action too. Note, that v4 has changed the semantics, we would specify `name: catalogs-cudo` now

Reply:
Done